sql/server.go

```diff
@@ -226,9 +270,10 @@ func (s *Server) exec(req driver.Request) (resp driver.Response, err error) {
 		if err = plan.Err(); err != nil {
 			return
 		}
-
 		resp.Results = append(resp.Results, result)
 	}
-
+	if err != nil {
+		panic(err)
```

Comment:
Ah, I see what this is doing now. Named return values in large methods like this are a bit opaque. My preference would be to rename the return value to `errDoNotUse` and then explicitly return the values we want to return everywhere. Failing that, wrap the err with another message: `panic(fmt.Errorf("unhandled error (this should never happen): %v", err))`